Service/utils.js

```diff
@@ -1,6 +1,5 @@
 const geoip = require("geoip-lite");
 const parser = require("ua-parser-js");
-const requestIp = require("request-ip");
```

Comment:
are you sure we dont need to keep this?

Reply:
yes, that was the thing that need the whole request